demo/Makefile

```diff
@@ -1,6 +1,6 @@
-TAG=$(shell date -I -u)
+TAG=$(shell date -u +"%Y-%m-%dT%H-%M-%S")
```

Comment:
No, this produces e.g.

    2017-01-12T19-11-56

whereas I wanted e.g.

    2017-01-12